README.md

```diff
@@ -1,8 +1,164 @@
-# sindri-scroll-sdk
+# Sindri Provers for Scroll SDK
 
-## Temporary Usage Instructions
 
-First, create your `config.json` file from a template
+<img src="./media/sindri-gradient-logo.webp" height="160" align="right"/>
+
+#### [Sindri Sign Up](https://sindri.app/signup) | [Scroll SDK Docs](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/) | [Getting Started](#getting-started) | [Development](#internal-development)
+
+Sindri provides automated ZK proving infrastructure, empowering hundreds of teams—including leading Layer 2s and rollups—to launch in minutes instead of months.
+Through the Sindri API, developers can seamlessly integrate verifiable computation, reducing time to market, cutting costs, and scaling faster.
+Sindri makes zero-knowledge infrastructure simple and accessible, facilitating automation within the most hardware-intensive layer of the ZK app deployment stack.
+
+This repository hosts a proving application image and Helm chart which will route all zkEVM proof generation to the Sindri platform for a Scroll SDK chain.
```

Comment:
"... for a Scroll SDK chain."

Is "Scroll SDK chain" a thing? Is this an accurate descriptor which will be clear to the ZK reader?

Reply:
This phrasing is modeled after the upstream Scroll docs, cf [Running a Scroll SDK devnet](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/devnet-deployment/).  (Devnet is a specific type of ephemeral chain used for developers to test out their setup or build applications on).  So I'm fairly confident that this language won't cause confusion

Reply:
Thanks for the 'splainin